test/feature/reverse_routing.feature

```diff
@@ -0,0 +1,31 @@
+Feature: reverse routing
+
+  Background:
+    Given an empty "controllers" directory
```

Comment:
Yeah, but exprestive throws an error if the controllers directory doesn't exist. That's intended functionality.

Since each test starts with an empty tmp dir there is no controllers directory so this creates one.